app/views/alchemy/base/500.html.erb

```diff
@@ -12,15 +12,29 @@
   ], search: false
 ) %>
 
+<% content_for(:alchemy_body_class) { 'error' } %>
+
 <%= render_message(:error) do %>
   <h1><%= Alchemy.t('An error happened') %></h1>
   <h2>
     <%= @error.class %>
     <%= @notice %>
   </h2>
-  <p>
+  <p class="hidden">
   <% @trace.each do |line| %>
     <%= line %><br>
   <% end %>
   </p>
+  <a href="#" class="error_details--toggle">
+    <%= Alchemy.t('Show error details') %>
+  </a>
 <% end %>
+
+<script>
+  $(function(){
+    $('.error_details--toggle').click(function(){
+      $('.error.message p').toggleClass('hidden');
+      return false;
+    });
+  });
+</script>
```

Comment:
I never use jQuery `.show()` and `.hide()` as they only toggle the `display` value. Maybe someday we want to change the `.hidden` class to use `opacity`. And I don't think adding `style` attributes directly on the element is ever a good idea. 

And I never ever use jQuery animated stuff. Because they don't use CSS animations, but JavaScript based animations and this is never a good idea. Browsers perform much better with CSS animations.